common-gradle-enterprise-bazel-configuration/README.md

```diff
@@ -0,0 +1,3 @@
+## Common Gradle Enterprise Bazel configuration
```

Comment:
"Gradle Enterprise" -> "Develocity" everywhere in this file

Reply:
This is a good point. We will do this wholistically across the entire repo after the next release. For now let's keep it consistent.